src/rules/trailingCommaRule.ts

```diff
@@ -56,7 +56,7 @@ const normalizeOptions = (options: Options): Options => {
             typeLiterals: options.multiline,
         };
     } else {
-        options.multiline = Object.assign({}, defaultOptions, options.multiline);
+        options.multiline = { ...defaultOptions, ...options.multiline };
```

Comment:
`options.multiline` can be undefined. Does typescript handle that case?
`option.singleline` can be undefined as well